abayo-luc/hello-books
# 164596009 User Registration

#### Description
Add user registration functionalities
#### Type of change
- [ ] Bug fix (change that would fix current implimention)
- [x] New feature (non-breaking changes)
- [ ] Breaking changes ( feature that would cause the current impliment break)
- [x] Changes that require documentation 

#### How Has This Been Tested?
- Clone the repo
- checkout `ft-user-registration-164596009`
- install dependencies `bundle install`
- run automated tests `bundle exc rspec`
- Test in postman `POST /api/v1/users`
    - [x] username
    - [ ] first_name
    - [ ] last_name
    - [x] phone_number
    - [x] password
    - [x] password_confirmation
- expect `response` with token and created user `instance`
#### Checklist:
 - [x] Follow the style guide
 - [x] Tested passes localy with the current changes
#### Pivotal Tracker ID
164596009
### Screenshots
<img width="1406" alt="Screen Shot 2019-03-19 at 14 23 41" src="https://user-images.githubusercontent.com/20681465/54605740-9dfeb580-4a52-11e9-8626-323c7dafd169.png">
